pkgs/hashicorp/terraform-ls/registry.yaml

```diff
@@ -1,14 +1,27 @@
 packages:
-  - type: github_release
+  - type: http
     repo_owner: hashicorp
     repo_name: terraform-ls
-    asset: terraform-ls_{{trimV .Version}}_{{.OS}}_{{.Arch}}.zip
     description: Terraform Language Server
+    url: https://releases.hashicorp.com/terraform-ls/{{trimV .Version}}/terraform-ls_{{trimV .Version}}_{{.OS}}_{{.Arch}}.zip
     checksum:
-      type: github_release
-      asset: terraform-ls_{{trimV .Version}}_SHA256SUMS
+      type: http
+      url: https://releases.hashicorp.com/terraform-ls/{{trimV .Version}}/terraform-ls_{{trimV .Version}}_SHA256SUMS
       file_format: regexp
       algorithm: sha256
       pattern:
         checksum: ^(\b[A-Fa-f0-9]{64}\b)
         file: "^\\b[A-Fa-f0-9]{64}\\b\\s+(\\S+)$"
+    version_constraint: semver(">= 0.30.2")
```

Comment:
Old versions can be downloaded from Release site, so `version_overrides` isn't needed.
https://releases.hashicorp.com/terraform-ls

Reply:
oh sorry, It certainly is. I will fix it.

Reply:
The following commit, don't use `version_constraint` and `version_overrides` and migrated to release site.
https://github.com/aquaproj/aqua-registry/pull/10085/commits/7f41ca3084e14d1bb6ff9cdc0f7fa2119d56fe8a

Reply:
Thanks.